normandy/recipes/tests/test_serializers.py

```diff
@@ -15,6 +15,7 @@ def test_it_works(self, rf):
         assert serializer.data == {
             'name': recipe.name,
             'id': recipe.id,
+            'last_updated': recipe.last_updated.replace(tzinfo=None).isoformat() + 'Z',
```

Comment:
Took a little extra magic and is not pretty but this seems to be the only way to format the date to match O_o